cmd/dep/init.go

```diff
@@ -196,31 +196,31 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, args []string) error {
 	// to generate the final lock memo.
 	s, err = gps.Prepare(params, sm)
 	if err != nil {
-		return errors.Wrap(err, "prepare solver")
+		return errors.Wrap(err, "init failed to recalculate the lock digest")
 	}
 
 	p.Lock.SolveMeta.InputsDigest = s.HashInputs()
 
 	// Pass timestamp (yyyyMMddHHmmss format) as suffix to backup name.
 	vendorbak, err := dep.BackupVendor(vpath, time.Now().Format("20060102150405"))
 	if err != nil {
-		return err
+		return errors.Wrap(err, "Manually backup vendor/ and retry the previous command. Failed to backup existing vendor directory")
```

Comment:
Flip these sentences?

Reply:
"failed to backup existing vendor directory. manually backup vendor/ and retry the previous command."

Reply:
Since this is a user facing message, I would prefer to display proper English, instead of sticking with the "errors shouldn't use punctuation or captial letters" pattern.

Here is what is printed with the code as-is:

> Manually backup vendor/ and retry the previous command. Failed to backup existing vendor directory: source is not a directory

and here is what it looks like when that error pattern is followed:

> failed to backup existing vendor directory. manually backup vendor/ and retry the previous command: source is not a directory

Reply:
Nevermind, I think this will work! 😀 

> init failed: first backup vendor/, delete it, and then retry the previous command: failed to backup existing vendor directory: source is not a directory